### ISSUE TITLE
Preview release doesnt see bundles in input/tests

### ISSUE THREAD

**Issue by citizenrich** [cqframework/vscode-cql#43]
The preview release of the plugin does not process bundles in input/tests. This breaks tests.

**Comment by mholck** [comment]
0.7.3 version of the plugin does only seem to work with tests that are individual resources and not bundles. Given all exports from MADiE are bundles it should either support bundles or there should be tooling to extract them into individual resources to facilitate testing

**Comment by brynrhodes** [comment]
Agreed, would be nice to have a "unbundle" feature so this could be easily done by users.

Right now that can be done using CQF-Tooling, see ExtractBundleResources:

https://github.com/cqframework/dqm-content-qicore-2025?tab=readme-ov-file#measure-package-prep-process
